tests/ocf-url_link_path_absolute/EPUB/package.opf

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<package xmlns="http://www.idpf.org/2007/opf" xmlns:epub="http://www.idpf.org/2007/ops" version="3.0" xml:lang="en" unique-identifier="pub-id">
+  <metadata xmlns:dc="http://purl.org/dc/elements/1.1/">
+    <dc:coverage>Open Container Format</dc:coverage>
+    <dc:creator>Ivan Herman</dc:creator>
+    <dc:date>2022-03-01</dc:date>
+    <dc:description>Use a path-absolute link from the content to a photograph. The folder hierarchy containing the photograph starts at the root level.</dc:description>
```

Comment:
For clarity, I would say "beginning with a leading slash" or similar.

Reply:
I added the term, but kept the official things as well, which refers back to the specification text.